cloud-providers/aws/beanstalk/payara-server-docker-aws-eb/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM payara/server-full
+
+COPY target/application.war $DEPLOY_DIR
+
+ENV DEPLOY_PROPS=--contextroot=/
+
+EXPOSE 8080
```

Comment:
Expose already done in _payara/server-full_ Docker Image, can be removed.

Reply:
I'd rather leave it here to stress that the port needs to be exposed for EB. It's redundant I know, but it's not documented that the Payara Docker image exposes 8080.